src/bin/stats.rs

```diff
@@ -1,3 +1,12 @@
+// Copyright (c) 2019, The rav1e contributors. All rights reserved
```

Comment:
The initial revision of `src/bin/stats.rs` was derived lines in `src/bin/common.rs`, dated 2018.
```suggestion
// Copyright (c) 2018-2019, The rav1e contributors. All rights reserved
```